### PR TITLE
Add container-structure-test

### PR DESCRIPTION
**Pull request by bells17**
https://github.com/GoogleContainerTools/container-structure-test

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -201,6 +209,17 @@ image-normal:
 image-with-sidecar:
 	docker buildx build --no-cache --load -t $(IMAGE_PREFIX)topolvm-with-sidecar:devel --build-arg TOPOLVM_VERSION=$(TOPOLVM_VERSION) --target topolvm-with-sidecar .
 
+.PHONY: container-stracture-test
```
Comment (pluser) [inline Makefile:212]:
```suggestion
.PHONY: container-structure-test
```

File: Makefile [hunk 2]
```diff
@@ -201,6 +209,17 @@ image-normal:
 image-with-sidecar:
 	docker buildx build --no-cache --load -t $(IMAGE_PREFIX)topolvm-with-sidecar:devel --build-arg TOPOLVM_VERSION=$(TOPOLVM_VERSION) --target topolvm-with-sidecar .
 
+.PHONY: container-structure-test
+container-stracture-test: ## Run container-structure-test.
```
Comment (daichimukai) [inline Makefile:213]:
```suggestion
container-structure-test: ## Run container-structure-test.
```

File: .github/workflows/main.yaml [hunk 3]
```diff
@@ -47,6 +47,24 @@ jobs:
...
+        image:
+          - "normal"
+          - "with-sidecar"
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - run: make install-container-structure-test
+      - run: make image-${{ matrix.image }}
+      - run: make container-stracture-test STRUCTURE_TEST_TARGET=${{ matrix.image }}
```
Comment (daichimukai) [inline .github/workflows/main.yaml:66]:
```suggestion
      - run: make container-structure-test STRUCTURE_TEST_TARGET=${{ matrix.image }}
```

**Review by daichimukai — CHANGES_REQUESTED** [review]
Please squash to one commit.

File: Makefile (hunk 1 above)
Comment (bells17) [reply Makefile]: I fixed it.

File: .github/workflows/main.yaml (hunk 3 above)
Comment (bells17) [reply .github/workflows/main.yaml]: I fixed it.

File: Makefile (hunk 2 above)
Comment (bells17) [reply Makefile]: I fixed it.

**Review by daichimukai — APPROVED** [review]

**Review by pluser — APPROVED** [review]

**Pull request merged**